docs/template-functions.md

```diff
@@ -0,0 +1,154 @@
+
+Renders the title of the current page:
+
+```
+<title>{{ render_title() }}</title>
+```
+
+You can override the title by setting a context variable called `title`:
+
+```
+{% with title = "foo" %}
+  {{ render_title() }}
+{% endwith %}
+```
+
+### `meta_description(description=None)`
+
+Renders the content of the meta description tag for the current page:
+
+```
+<meta name="description" value="{{ get_meta_description() }}">
+```
+
+You can override the meta description by setting a context variable called `meta_description`. You might want to use this in, e.g. the `get_context_data` method of a function to set a default for models that do not inherit from SearchMetaBase:
+
+```
+{% with meta_description = 'foo' %}
+  <meta name="description" content="{{ get_meta_description() }}">
+{% endwith %}
+```
+
+### `get_meta_robots(context, index=None, follow=None, archive=None)`
+
+Renders the content of the meta robots tag for the current page:
+
+```
+<meta name="robots" content="{{ get_meta_robots() }}">
+```
+
+You can override the meta robots by setting boolean context variables called
+``robots_index``, ``robots_archive`` and ``robots_follow``:
+
+```
+{% with robots_follow = 1 %}
+  {{ get_meta_robots() }}
+{% endwith %}
+```
+
+You can also provide the meta robots as three boolean arguments to this
+tag in the order 'index', 'follow' and 'archive':
+
+```
+{{ get_meta_robots(1, 1, 1) }}
+```
+
+## Navigation functions
+### `render_navigation(pages, section=None)`
+
+Renders the site navigation using the template specified at ``pages/navigation.html``. By default this is just an unordered list with each navigation item as a list item.  The simplest usage is like this:
+
+```
+{{ render_navigation(pages.homepage.navigation) }}
+```
+
+Which would produce an output like this:
+
+```
+<ul>
+  <li>
+    <a href="/page-1/">Page One</a>
+  </li>
+
+  <li>
+    <a href="/page-2/">Page Two</a>
+  </li>
+</ul>
+```
+
+If you would like the "base page" (the page that the navigation is being based off) to be included in the navigation simply add the ``section`` kwarg:
+
+```
+{{ render_navigation(pages.homepage.navigation, section=pages.homepage) }}
+```
+
+The output of this would be:
+
+```
+<ul>
+  <li>
+    <a class="here" href="/">Homepage</a>
+  </li>
+
+  <li>
+    <a href="/page-1/">Page One</a>
+  </li>
+
+  <li>
+    <a href="/page-2/">Page Two</a>
+  </li>
+</ul>
+```
+
+You will probably want to override the output in your project. For example, you'll almost certainly want to add class names, and to handle children of children. We suggest you make a local copy of `apps/pages/templates/pages/navigation.html` in your project.
+
+### `render_breadcrumbs(page=None, extended=False)`
+
+Renders the breadcrumbs trail for the current page:
+
+```
+{{ render_breadcrumbs() }}
+```
+
+To override and extend the breadcrumb trail within page applications, add the ``extended`` flag to the tag and add your own breadcrumbs underneath:
```

Comment:
double backticks

Reply:
Also your example below isn't super clea on what you mean by `add your own breadcrumbs underneath`